lib/backend/etcdbk/etcd.go

```diff
@@ -325,17 +325,17 @@ func (b *EtcdBackend) reconnect(ctx context.Context) error {
 		if err != nil {
```

Comment:
Can't comment on the line above (324) but with these changes `caCertPEM` doesn't really need to be declared outside the "if" anymore, you can just do `caCertPEM, err := ioutil.ReadFile(...)` (which would get rid of `var err error` as well).